garden-backend-service/src/api/dependencies/database.py

```diff
@@ -0,0 +1,20 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.asyncio import AsyncSession, create_async_engine
+from src.config import get_settings
+
+postgres_url = get_settings().SQLALCHEMY_DATABASE_URL
+engine = create_engine(postgres_url, echo=True)
+SessionLocal = sessionmaker(bind=engine)
+
+# lifted from funcx web service in case we decide we need it
+_async_url = postgres_url.replace("://", "+asyncpg://")
+_async_engine = create_async_engine(_async_url, echo=True)
+_SessionLocalAsync = sessionmaker(bind=_async_engine, class_=AsyncSession)
```

Comment:
Have you played with this? Do we want to be using it by default or are there some cases where we want the sync session over the sync session?